velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -77,6 +77,20 @@ DEFINE_bool(
 
 namespace facebook::velox::exec::test {
 
+bool AggregationFuzzerBase::isSupportedType(const TypePtr& type) const {
+  if (type->isDate() || type->isIntervalDayTime() || type->isUnKnown()) {
```

Comment:
I am wondering whether I should make this a property of referenceQueryRunner.

Reply:
I think ddb doesnt support Timestamp (because of nanosecond precision), and these types arent supported in PQR; so I still feel at some level this should be properties of respective query runner. Let me know what you think.